add docs for dove.toml manifest file (#3)
pontem-network/docs: toolset/dove_toml_manifest.md

# `Dove.toml` manifest format

`Dove.toml` is a [TOML](https://toml.io/en/) file which describes properties of your Move project.

It consists of the following sections: 

* [`[package]`](#the-package-section) 
    * `name` - The name of the project
      
    * [`account_address`](#the-account_address-field) - The user account address

    * [`dialect`](#the-dialect-field) - Address format of the project 
      

>     * [`dependencies`](#the-dependencies-field) - Specifies sets of project dependencies

Reviewer: ```suggestion
    * [`dependencies`](#the-dependencies-field) - Specifies a set of project dependencies
```
or 
```suggestion
    * [`dependencies`](#the-dependencies-field) - Specifies project dependencies
```

    
* [`[layout]`](#the-layout-section) - Directory layout of the project
    * `modules_dir`
    * `scripts_dir`
    * `tests_dir`
    
## The `[package]` section

```toml
[package]
name = "my_project"
account_address = "<your account address>"
dialect = "polkadot"
dependencies = [
    { git = "https://github.com/pontem-network/move-stdlib", branch = "v1.2.0" },
    { path = "./local_modules" }
]
```

### The `account_address` field

The address of the user account. 

> 1. You can add `{{sender}}` clause to your source code, 

Reviewer: ```suggestion
1. You can use `{{sender}}` clause anywhere in your source code, 
```

   which will be replaced with the value of `account_address` at compilation time.
    
2. If you `dove run` the script without `--signers` flag, 

>    this address will be a default signer passed to script function.  

Reviewer: ```suggestion
   this address will be the default `signer` passed to the script function.  
```

> 3. If you don't wrap your module in the `address {}` block, 

Reviewer: ```suggestion
3. If you don't wrap your module with the `address {}` block, 
```

>    this is the address where that module will be compiled at. 

Reviewer: ```suggestion
   this value is used as the address of the module. 
```
